verifyFiles.py

```diff
@@ -193,13 +198,38 @@ def pretty_string(diff_dict,conditions_dict):
         output_string+="\n"
     return output_string
 
+#Method is_subject_folders_same checks if the subject_folders under different conditions are the same. If not , it stops the execution of the script.
```

Comment:
The method actually checks if the subjects in the *first* condition are present in every other condition. But it's not able to detect that a subject is missing in the first condition. That's ok for now, but your method is overly complex. Based on the example of `common_paths_list`, you could write it in a few lines only, using `conditions_list` only (no call to the file system).